packages/app/src/index.css

```
@@ -78,36 +78,53 @@
+    --shell-border-weak: #E3DED6;
+    --shell-highlight-overlay: radial-gradient(circle at top, rgb(255 255 255 / 0.72), transparent 52%);
     --shell-frame-border: color-mix(in srgb, var(--shell-border-base) 78%, #c4b5a8 22%);
     --shell-frame-shadow:
       0 24px 64px -36px rgb(33 26 22 / 0.34),
       0 10px 28px -20px rgb(33 26 22 / 0.2);
     --shell-frame-surface: linear-gradient(180deg, var(--shell-surface-strong) 0%, var(--shell-surface-base) 100%);
     --shell-titlebar-border: color-mix(in srgb, var(--shell-border-weak) 72%, var(--shell-border-base) 28%);
     background:
-      radial-gradient(circle at top, rgb(255 255 255 / 0.72), transparent 52%),
+      var(--shell-highlight-overlay),
       linear-gradient(180deg, var(--shell-background-base), var(--shell-background-weak));
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)